utils.py

```diff
@@ -35,6 +38,48 @@
 DAYS = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"]
 WEEKEND_IDS = ["1", "7", "8", "14"]
 
+@dataclass
+class ModelParams:
```

Comment:
I don't have a strong opinion on this, I can see pros and cons to each

Reply:
This might be a pain but I do like how MVRP uses a class as seen in this file https://github.com/dwave-examples/mvrp/blob/main/solver/cvrp.py also this file https://github.com/dwave-examples/mvrp/blob/main/solver/solver.py
But this might be scope creep/overkill.

I do like the way everything is labeled with Jameson's solution (I was updating the tests in my branch and it was a pain without knowing which variable was which) but I agree that as it currently stands it's kind of hiding the parameters. I would say at the bare minimum it would be good to have keyword arguments or another way to label the parameters.

If either of you have other suggestions let me know!

Reply:
A middle ground between @jlalbers 's approach and what's already there would be
```python
from dataclasses import dataclass, asdict

@dataclass
class ModelParams:
    param1: int = 1
    param2: str = "hi there"

def func(param1, param2):
    print(param1, param2)

func(**asdict(ModelParams()))
```

it could make sense to do ^ for this PR, then make the deeper class restructure as a followup?

Reply:
Probably the best balance between function readability and ease of defining parameters. Will rework this in a new commit

Reply:
Sounds good! Thank you!

Reply:
Updated the example to implement @arcondello 's strategy